src/integrationTest/java/uk/gov/hmcts/darts/arm/service/AbstractArmBatchProcessResponseFilesIntTest.java

```diff
@@ -542,13 +544,15 @@ void batchProcessResponseFiles_successful_withUploadFileAndCreateRecordFile() th
 
         when(armDataManagementApi.listResponseBlobs(hashcode1)).thenReturn(hashcodeResponses);
 
+        String inputUploadFileTest1 = "tests/arm/service/ArmBatchResponseFilesProcessorTest/ValidResponses/InputUploadFile.rsp";
         String createRecordFileTest1 = "tests/arm/service/ArmBatchResponseFilesProcessorTest/ValidResponses/CreateRecord.rsp";
         String validUploadFileTest1 = "tests/arm/service/ArmBatchResponseFilesProcessorTest/ValidResponses/UploadFile.rsp";
 
+        BinaryData inputUploadFileBinaryDataTest1 = convertStringToBinaryData(getInputUploadFileContents(inputUploadFileTest1));
         BinaryData createRecordBinaryDataTest1 = convertStringToBinaryData(getCreateRecordFileContents(createRecordFileTest1, armEod1.getId()));
         BinaryData uploadFileBinaryDataTest1 = convertStringToBinaryData(getUploadFileContents(validUploadFileTest1, armEod1.getId(), media1.getChecksum()));
 
-
+        when(armDataManagementApi.getBlobData(blobNameAndPath1)).thenReturn(inputUploadFileBinaryDataTest1);
```

Comment:
As discussed previously, we shouldn't be mocking within Integration Tests the classes that aren't the exit point from DARTS, including `armDataManagementApi`. However, I realise that this wasn't introduced as part of this ticket. I've spoken to @hemantasharma1129 and agreed this can be refactored as part of a separate ticket.